Bug 1210304 - David's comment about number of columns in JSON

MariaKolesina opened the pull request:
David's comment:
"Given that the number of columns will not be configurable, I think that it doesn't need to be in this JSON file anymore. Maybe find the code that reads this file and just define a const for the number of columns."
Removed number of columns from default-homescreens.json. Define a const for the number of columns in configurator.js

MariaKolesina commented:
@harman-red-square/harman-reviewers

ruvsmirnov commented on apps/verticalhome/js/configurator.js:
```diff
@@ -20,6 +20,8 @@
   var singleVariantApps = {};
   var simPresentOnFirstBoot = true;
 
+  var DEFAULT_COLUMNS = 2;
```
May be it is better to  use const instead var?
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/const

MariaKolesina replied:
I can change it to const, but this declaration like var  is consistent with existing code in this file

ruvsmirnov replied:
It is a minor issue. But it is better to ban change this variable.

egorlevichev commented on apps/verticalhome/index.html:
```diff
@@ -68,50 +73,56 @@
+    <script defer src="js/clock.js"></script>
     <script defer src="js/app.js"></script>
-    <script defer src="js/search.js"></script>
+<!--    <script defer src="js/search.js"></script> -->
```
do we need 'search.js' script in verticalhome app? If no, then it's better to delete this line instead of commenting, since commented scripts will be packaged into 'application.zip' archive and will be available for loading from other scripts. (AFAIR so please correct me if I'm wrong)

MariaKolesina committed:
Bug 1210304 - David's comment about number of columns in JSON

David's comment:
"Given that the number of columns will not be configurable, I think that it doesn't need to be in this JSON file anymore. Maybe find the code that reads this file and just define a const for the number of columns."
Removed number of columns from default-homescreens.json. Define a const for the number of columns in configurator.js
